src/univariate/continuous/normal.jl

```diff
@@ -76,14 +76,23 @@ entropy(d::Normal) = (log2π + 1)/2 + log(d.σ)
 #### Evaluation
 
 # Helpers
+"""
+    xval(d::Normal, z::Real)
```

Comment:
inconsistent hyphenation

Reply:
Could you elaborate? It seems the project-wide style is a four space on first line, skip line and start without indentation from inspecting various files.

Reply:
„x value“ versus „z-value“